### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by tomaszaba**
Hi Ernest, 

Please review this PR.
I addressed issues #2, #3, #4 , #5 , #6 , #7 , #8 , #9 , #11 , #12 , #13. 

Regarding issue #10 , I was not sure where to put it, so I left where it was before. Please feel free to move where you find it fit.

On #7, the flowchart does not get centralised when it gets uploaded to GitHub. Locally, it got centralised after I reduced the width parameters. See below: 

image

Please note that this time I am using `quarto`. 

Regards.

### CODE REVIEW

File: README.qmd [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+In this way, more than just an R-based implementation of the ENA for SMART software, mwana’s key added value lies in its ability to simplify the above alluded cumbersome workflow into a wholesome experience, all in one place. This is especially beneficial when handling large datasets, a day-to-day practice at IPC.
```
Comment (ernestguevarra) [inline README.qmd:34]: I suggest re-phrasing this section to the following:

```md
`mwana` was borne out of the author’s own experience of having to work with multiple child anthropometric datasets to conduct data quality assessment and prevalence estimation on as part of the data quality assurance team of the Integrated Phase Classification (IPC). The current standard child anthropometric data appraisal workflow is extremely cumbersome requiring significant time and effort utilising different software tools (SPSS, Excel, Emergency Nutrition Assessment or ENA software) for each step of the process for a single dataset. This process is repeated for every dataset needing to be processed and often needing to be implemented in a relatively short period time. This manual and repetitive process, by its nature, is extremely error-prone.

`mwana`, which is primarily a an R-based implementation of the ENA for SMART software, simplifies this cumbersome workflow into a programmable process particularly when handling large multiple datasets.
```

**Review by ernestguevarra — CHANGES_REQUESTED** [review]
@tomaszaba, great work on this!

See my specific comments above. Please address them on the same branch. When you push your changes, they will go to this same pull request and then I can do a final review.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,28 +1,19 @@
 Type: Package
 Package: ipccheckr
-Title: Toolkit for Performing IPC Acute Malnutrition-related Data Checks
+Title: Utilities for analysing children's nutritional status
```
[the remark this answers is not in the document]
Comment (tomaszaba) [reply DESCRIPTION:3]: Thanks for this review. I skimmed the rules through, then I checked some package titles on GitHub. Found that the major error in this title was probably not capitalised like a title. I have revised this to the following: "Utilities for Analysing Children's Nutritional Status". I have to admit though that I am not quite happy with my title. Do you have any suggestions?

File: DESCRIPTION [hunk 3]
```diff
@@ -1,28 +1,19 @@
...
 Package: ipccheckr
-Title: Toolkit for Performing IPC Acute Malnutrition-related Data Checks
+Title: Utilities for analysing children's nutritional status 
 Version: 0.0.0.9000
 Authors@R: c(
     person("Tomás", "Zaba", , "tomas.zaba@outlook.com", role = c("aut", "cre", "cph"),
            comment = c(ORCID = "0000-0002-7079-3574")),
     person("Ernest", "Guevarra", role = c("aut", "cph"),
-           comment = c(ORCID = "0000-0002-4887-4415")),
-    person("Douglas", "Jayasekaran", role = "ctb")
```
[the remark this answers is not in the document]
Comment (tomaszaba) [reply DESCRIPTION:10]: We will be able to ascertain the type of his contribution once I share the link with him to review the documentation.I expect his reviews on the Vignettes, other than the function documentation per se. We can revisit this later.

File: DESCRIPTION [hunk 4]
```diff
@@ -1,28 +1,19 @@
...
-    Overview (HNO) of the Humanitarian Programme Cycle (HPC). The accepted
-    sources of evidence on prevalence of acute malnutrition varies from
-    surveys, community-based screenings and sent- inel sites. According to
-    the IPC AMN protocols, every evidence must be subject to quality
-    checks to ascertain their reliability and decided whether to use or
-    not. This package provides useful functions and utilities that
-    supports IPC AMN-related data analysis check.
+Description: A streamlined and comprehensive implementation of SMART Methodology
+  guidelines for data quality checks and prevalence analyses, with enhanced 
+  efficiency for handling large datasets.
```
[the remark this answers is not in the document]
Comment (tomaszaba) [reply DESCRIPTION:13]: Yes, this 
> "But, I think what you are saying here is more in terms of the package helping to streamline the checks and estimation process such that multiple datasets that need checking and analysing (which I think is what you mean by large datasets) can be done through a streamlined, reproducible process. " 

is correct about what I meant. It is also on the fact that using ENA, one needs to repeat the workflow X times, depending on the number of provinces, districts in the dataset, as the units of analysis. How do you suggest we clear this?

File: DESCRIPTION [hunk 5]
```diff
@@ -1,28 +1,19 @@
...
 License: GPL (>= 3)
-URL: https://github.com/tomaszaba/ipccheckr
-BugReports: https://github.com/tomaszaba/ipccheckr/issues
+URL: https://github.com/nutriverse/mwana
```
[the remark this answers is not in the document]
Comment (tomaszaba) [reply DESCRIPTION:15]: Done.

File: README.md [hunk 6]
```diff
@@ -1,136 +1,143 @@
 
+
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# `ipccheckr`:Toolkit for Performing IPC Acute Malnutrition-related Data Checks
+# `mwana`: Utilities for analysing children’s nutritional status
```
[the remark this answers is not in the document]
Comment (tomaszaba) [reply README.md:5]: Done.

File: README.qmd [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+library(ipccheckr)
+```
+
+# `mwana`: Utilities for analysing children's nutritional status 
+
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/nutriverse/mwana/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/tomaszaba/ipccheckr/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/nutriverse/mwana/branch/main/graph/badge.svg)](https://app.codecov.io/gh/tomaszaba/ipccheckr?branch=main)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
```
[the remark this answers is not in the document]
Comment (tomaszaba) [reply README.qmd:25]: Done.

File: DESCRIPTION (hunk 4 above)
[the remark this answers is not in the document]
Comment (tomaszaba) [reply DESCRIPTION:13]: Here is the revised text: 

"A streamlined and comprehensive implementation of the Standardized
  Monitoring and Assessment of Relief and Transition (SMART) Methodology
  <https://smartmethodology.org/> guidelines for data quality checks and 
  prevalence analyses, with enhanced programmable process particularly when 
  handling large multiple datasets." 

I used your text on "programmable process particularly when handling large multiple datasets" from the Motivation edition you made. Does this make the description less ambiguous?

**Review by ernestguevarra — APPROVED** [review]
@tomaszaba, all this looks good! Well done! You can merge this to main now and go on to the next task.

**Pull request merged**